pkg/cmdutil/userio/wizard/wizard.go

```diff
@@ -177,21 +198,18 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		return m, tea.Sequence(updateListener, cmd)
 	case tea.KeyMsg:
 		log.Debug().Msgf("Wizard: Received keystroke [%s]", msg.String())
-
 		switch msg.Type {
 		case tea.KeyCtrlC:
-			m.quitting = true
-			return m, tea.Quit
-		default:
 			if m.inputModel != nil {
-				newInputModel, inputCmd := m.inputModel.Update(msg)
+				newInputModel, inputCmd = m.inputModel.Update(msg)
 				m.inputModel = newInputModel
 				return m, inputCmd
 			}
+			m.quitting = true
+			return m, tea.Quit
```

Comment:
Suggestion: should we pass a doneMsg instead, so the logic with completing all the hanging messages will also execute?

Reply:
For a <kbd>ctrl</kbd>+<kbd>c</kbd> I think we just want to exit regardless of anything in the message queue